recipes/tjpcov/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "tjpcov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tjpcov-{{ version }}.tar.gz
+  sha256: fd4cd437105521e11c7f977d522cdfae75fed6d7e0c06afa599cc7eec8d7a4eb
+
+build:
+  noarch: python
+  script: python -m pip install . -vv --no-deps
```

Due to a bug in multiple outputs we do need the no build isolation there.

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

---

I have an issue where when I try to use the environment variable for {{ PYTHON }} I get an exception that the variable is empty / undefined:

```
Error: Failed to render jinja template in /Users/matt/Repos/tjpcov-feedstock/recipes/tjpcov/meta.yaml:
'PYTHON' is undefined
```

This was happening on the build server as well, not just my local machine.  Any ideas why this is happening?  I swapped to just using python because I wasn't sure.